### PR TITLE
add: sql optimizer

### PR DESCRIPTION
**Pull request by ArthurFlag**

### CODE REVIEW

File: static/includes/faq-ai.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+To address the possibility of slow query logs containing sensitive data, Aiven
+offers data masking capabilities that replaces sensitive parameters within
+queries with question marks (`?`). Data masking is enabled by default.
+
+:::note
+The masking option is not available for
+the Standalone SQL query optimizer yet.
+:::
+
+**Is there an option to ask Aiven not to keep my SQL queries?**
```
Comment (Voded) [inline static/includes/faq-ai.md:19]: @ArthurFlag - remove this question, as we are working on a new checkbox that will allow the user to select 'don't save my query'. you can maybe add this specific question in 3-4 weeks when feature will go live.
(jira: https://aiven.atlassian.net/browse/EVERSQL-418)

**Review by Voded — COMMENTED** [review]
see my comment (all the rest looks great). thanks

**Review by Voded — APPROVED** [review]

File: docs/tools/query-optimizer.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+1. Select your database type and version.
+
+   :::note
+   The SQL query optimizer supports PostgreSQL® and MySQL®.
+   :::
+1. Paste your query and click **Next**.
+1. Optional:
+   1. Provide your table structure and statistics by running the query provided in
+      the UI.
+   1. Past it in the **Query output** field.
```
Comment (staceysalamon-aiven) [inline docs/tools/query-optimizer.md:30]:
```suggestion
   1. Paste it in the **Query output** field.
```

File: docs/tools/query-optimizer.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+1. Click **Optimize**.
+
+The optimation report shows the optimized query and potential optimal indexes.
```
Comment (staceysalamon-aiven) [inline docs/tools/query-optimizer.md:33]:
```suggestion
The optimization report shows the optimized query and potential optimal indexes.
```

File: static/includes/faq-ai.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+**Does Aiven AI Optimizer mask/obfuscate my queries?**
+
+Yes, Aiven AI Optimized provides a non-intrusive solution to optimize your
```
Comment (staceysalamon-aiven) [inline static/includes/faq-ai.md:3]:
```suggestion
Yes, Aiven AI Optimizer provides a non-intrusive solution to optimize your
```

File: static/includes/faq-ai.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+database performance without compromising sensitive data access.
+It achieves this by gathering information on schema structure, database statistics, and
+other signals to detect potential performance problems and offer optimization
+recommendations, without requiring credentials or access to the actual data in
+the database.
+
+To address the possibility of slow query logs containing sensitive data, Aiven
+offers data masking capabilities that replaces sensitive parameters within
```
Comment (staceysalamon-aiven) [inline static/includes/faq-ai.md:11]: Or maybe just: "offers data masking that replaces..."
```suggestion
offers data masking capabilities that replace sensitive parameters within
```

File: docs/tools/query-optimizer.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+:::
+
+To optimize a query:
+
+1. Click **Tools** > **SQL query optimizer**.
+1. Click **Optimize a query**.
+1. Select your database type and version.
+
+   :::note
+   The SQL query optimizer supports PostgreSQL® and MySQL®.
```
Comment (staceysalamon-aiven) [inline docs/tools/query-optimizer.md:24]: This could maybe be in the introduction instead since the options are shown in the UI?

**Review by staceysalamon-aiven — APPROVED** [review]
LGTM! Just fixed a couple of typos and added one minor stylistic suggestion.

**Pull request merged**